scripts/run_tests.py

```diff
@@ -66,7 +81,6 @@ def run_narrative():
             break
         if "is already in use" in line:
             os.killpg(os.getpgid(nb_server.pid), signal.SIGTERM)
```

Comment:
Will this kill a running docker container, too? Or will that container just bomb out if it's on an already-used port?

Reply:
It only kills the process that the script started, not anything else. If you already have something running on port 32323, the script generously allows it to live.